pkg/gui/context.go

```diff
@@ -391,6 +391,7 @@ func (gui *Gui) switchContextToView(viewName string) error {
 	return gui.switchContext(gui.State.ViewContextMap[viewName])
 }
 
+// TODO: add documentation about this function
 func (gui *Gui) returnFromContext() error {
```

Comment:
sorry that this wasn't already documented: this function's job is to pop the current context off the context stack: deactivating it and activating the next context in the stack